mgr/src/server/plugins/config_snapshots.cr

```diff
@@ -54,6 +56,27 @@ delete "/api/v1/config-snapshots/:snap_name" do |env|
   env.response.status_code = 204
 end
 
-get "/api/v1/config-snapshots" do
+get "/api/v1/config-snapshots" do |env|
+  snap_dir = "#{GlobalConfig.workdir}/config-snapshots"
+
+  unless Dir.exists?(snap_dir)
+    Dir.mkdir_p snap_dir
+  end
+
+  env.response.status_code = 200
+
   Datastore.list_config_snapshots.to_json
 end
+
+get "/api/v1/config-snapshots/:snap_name" do |env|
+  snap_name = env.params.url["snap_name"]
+  snap_dir = "#{GlobalConfig.workdir}/config-snapshots"
+
+  unless Dir.exists?(snap_dir)
+    Dir.mkdir_p snap_dir
```

Comment:
Do not create directory in Get API call. If directory not exists, then ERROR.

Reply:
done